src/__tests__/mongo.test.ts

```diff
@@ -80,26 +103,71 @@ describe("connectToDatabaseViaEnvVar", () => {
 
     process.env.MONGODB_URI = "mongodb://127.0.0.1:6666";
     try {
-      await connectToDatabaseViaEnvVar({ssl: false, tlsCAFile: "", serverSelectionTimeoutMS: 100}, logger);
+      await connectToDatabaseViaEnvVar(
+        { ssl: false, tlsCAFile: "", serverSelectionTimeoutMS: 100 },
+        logger
+      );
     } catch (error: any) {
       expect(error["message"]).toEqual("connect ECONNREFUSED 127.0.0.1:6666");
     }
 
     const errorLog = logOutput.find((log) => log.includes("ENGEXPUTILS008"))!;
-    expect(JSON.parse(errorLog)["error"]).toEqual("MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:6666");
+    expect(JSON.parse(errorLog)["error"]).toEqual(
+      "MongooseServerSelectionError: connect ECONNREFUSED 127.0.0.1:6666"
+    );
   });
 
   it("should default to the LambdaLogger if no logger is provided", async () => {
-    jest.spyOn(LambdaLogger.prototype, "error");
+    vi.spyOn(LambdaLogger.prototype, "error");
 
     delete process.env.MONGODB_URI;
 
     try {
-      await connectToDatabaseViaEnvVar({ssl: false, tlsCAFile: "", serverSelectionTimeoutMS: 0});
+      await connectToDatabaseViaEnvVar({
+        ssl: false,
+        tlsCAFile: "",
+        serverSelectionTimeoutMS: 0,
+      });
     } catch (error: any) {
       expect(error.message).toEqual("MONGODB_URI environment variable not set");
     }
 
-    expect(LambdaLogger.prototype.error).toHaveBeenCalledWith("ENGEXPUTILS006", { error: "MONGODB_URI environment variable not set" });
+    expect(LambdaLogger.prototype.error).toHaveBeenCalledWith(
+      "ENGEXPUTILS006",
+      { error: "MONGODB_URI environment variable not set" }
+    );
+  });
+});
+
+describe("disconnectFromDatabase", () => {
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+  it("should disconnect from the database and log", async () => {
+    const logger: ILog = {
+      info: vi.fn(),
+      error: vi.fn(),
+    };
```

Comment:
We can push this up to the `describe` scope, we shouldn't need to define it in both tests.